metadata-core/src/main/java/com/hashmap/haf/metadata/core/data/resource/jdbc/model/JdbcResource.java

```diff
@@ -47,13 +56,64 @@ public void setPassword(String password) {
     }
 
     @Override
-    public void push() {
+    public void push(Map payload) {
+        //TODO : Will be implemented later, when we have JDBC as Sink for Metadata
+    }
 
+    @Override
+    public Map pull(String query) throws Exception {
+        Map<String, Object> payload = new HashMap<>();
+        Connection connection = null;
+        Statement statement = null;
+        try {
```

Comment:
you could have used try-with-resources here.
Also, there should be support of db connection pooling which will control db connections.
You can think about Spring JDBC module instead of plain jdbc